src/Application/Cli/PreviewCommand.php

```diff
@@ -78,10 +77,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (!$this->isFound($input->getOption('livereload'))) {
-            $output->writeln('<error>Impossible to launch Livereload, did you forgot to install it with "pip install livereload" (sudo maybe required)?</error>');
+        if ($input->getOption('livereload')) {
+            if ($input->getOption('livereload') === null) {
+                $input->setOption('livereload', 'livereload');
+            }
```

Comment:
What I mean is that:

```php
if ($input->getOption('livereload')) {
    if ($input->getOption('livereload') === null) {
```

The second `if` will never be used because `$input->getOption('livereload')` is already not null (checked by the first `if`). So I don't see how we can ever enter the second if.